features/bootstrap/FeatureContext.php

```diff
@@ -194,8 +222,36 @@ public function proc( $command, $assoc_args = array(), $path = '' ) {
 			$env['WP_CLI_CACHE_DIR'] = $this->variables['SUITE_CACHE_DIR'];
 		}
 
-		$path = "{$this->variables['RUN_DIR']}/{$path}";
-		return Process::create( $command, $path, $env );
+		if ( isset( $this->variables['RUN_DIR'] ) ) {
+			$cwd = "{$this->variables['RUN_DIR']}/{$path}";
+		} else {
+			$cwd = null;
+		}
+
+		return Process::create( $command, $cwd, $env );
+	}
+
+	/**
+	 * Start a background process. Will automatically be closed when the tests finish.
```

Comment:
The main idea behind `\WP_CLI\Procees` is to make it easy to capture all the output from the process in a structured format.

For background processes, we're not always interested in output (and it's trickier to capture).